python/lsst/afw/geom/_transform.cc

```diff
@@ -0,0 +1,142 @@
+                cls.def(py::init<ast::Mapping const &, bool>(), "mapping"_a, "simplify"_a = true);
+
+                cls.def_property_readonly("hasForward", &Class::hasForward);
+                cls.def_property_readonly("hasInverse", &Class::hasInverse);
+                cls.def_property_readonly("fromEndpoint", &Class::getFromEndpoint);
+                cls.def_property_readonly("toEndpoint", &Class::getToEndpoint);
+
+                // Return a copy of the contained Mapping in order to assure changing the returned Mapping
+                // will not affect the contained Mapping (since Python ignores constness)
+                cls.def("getMapping", [](Class const &self) { return self.getMapping()->copy(); });
+
+                cls.def("applyForward",
+                        py::overload_cast<FromArray const &>(&Class::applyForward, py::const_), "array"_a);
+                cls.def("applyForward",
+                        py::overload_cast<FromPoint const &>(&Class::applyForward, py::const_), "point"_a);
+                cls.def("applyInverse", py::overload_cast<ToArray const &>(&Class::applyInverse, py::const_),
+                        "array"_a);
+                cls.def("applyInverse", py::overload_cast<ToPoint const &>(&Class::applyInverse, py::const_),
+                        "point"_a);
+                cls.def("inverted", &Class::inverted);
+                /* Need some extra handling of ndarray return type in Python to prevent dimensions
+                 * of length 1 from being deleted */
+                cls.def("_getJacobian", &Class::getJacobian);
+                // Do not wrap getShortClassName because it returns the name of the class;
+                // use `<class>.__name__` or `type(<instance>).__name__` instead.
+                // Do not wrap readStream or writeStream because C++ streams are not easy to wrap.
+                cls.def_static("readString", &Class::readString);
+                cls.def("writeString", &Class::writeString);
+
+                declareMethodTemplates<FromEndpoint, ToEndpoint, GenericEndpoint>(cls);
+                declareMethodTemplates<FromEndpoint, ToEndpoint, Point2Endpoint>(cls);
+                declareMethodTemplates<FromEndpoint, ToEndpoint, SpherePointEndpoint>(cls);
+
+                // str(self) = "<Python class name>[<nIn>-><nOut>]"
+                cls.def("__str__", [pyClassName](Class const &self) { return formatStr(self, pyClassName); });
+                // repr(self) = "lsst.afw.geom.<Python class name>[<nIn>-><nOut>]"
+                cls.def("__repr__", [pyClassName](Class const &self) {
+                    return "lsst.afw.geom." + formatStr(self, pyClassName);
+                });
+
+                table::io::python::addPersistableMethods<Class>(cls);
+            });
+}
+}  // namespace
+void wrapTransform(lsst::utils::python::WrapperCollection &wrappers) {
+    wrappers.addSignatureDependency("lsst.afw.typehandling");
```

Comment:
Does the `Transform` class here inherit from `typehandling::Storable` in C++?  Usually that's why one would add this dependency, but it'd be an inheritance dependency, not just a signature one, and I don't see the inheritance declared in the class wrappers.